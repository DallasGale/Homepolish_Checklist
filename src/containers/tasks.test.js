import React from 'react';
import Tasks from './tasks.container';
import renderer from 'react-test-renderer';
import 'jest-styled-components'

const tasks = [];

describe('<Tasks />', () => {
    it('should render without exploding!', () => {
        const tree = renderer.create(<Tasks tasks={ tasks } />).toJSON()
        expect(tree).toMatchSnapshot();
    })
})