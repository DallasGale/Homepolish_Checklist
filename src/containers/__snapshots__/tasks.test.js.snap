// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tasks /> should render without exploding! 1`] = `
.c0 {
  background: #fafafa;
  max-width: 510px;
  margin: 0 auto;
  width: 100%;
  position: relative;
}

.c1 {
  padding: 1em 1.3em 0 1.3em;
}

.c1:last-child {
  padding-bottom: 1em;
}

.c2 {
  border-bottom: 2px solid #e9e9e9;
  color: #2b2b2b;
  font-size: 0.85em;
  margin-bottom: 0;
  padding-left: 0.8em;
  padding-bottom: 1em;
  -webkit-letter-spacing: 0.2em;
  -moz-letter-spacing: 0.2em;
  -ms-letter-spacing: 0.2em;
  letter-spacing: 0.2em;
  text-transform: uppercase;
}

.c4 {
  border-bottom: 2px solid #e9e9e9;
  color: grey;
  font-size: 0.85em;
  margin-bottom: 0;
  padding-left: 0.8em;
  padding-bottom: 1em;
  -webkit-letter-spacing: 0.2em;
  -moz-letter-spacing: 0.2em;
  -ms-letter-spacing: 0.2em;
  letter-spacing: 0.2em;
  text-transform: uppercase;
}

.c3 {
  padding-left: 0;
  padding-top: 0;
  margin-top: 0;
}

<div
  className="c0"
>
  
  <div
    className="c1"
  >
    <h2
      className="c2"
      color={undefined}
    >
      
                                No Todo's
    </h2>
    <ul
      className="c3"
    />
  </div>
  <div
    className="c1"
  >
    <h2
      className="c4"
      color="grey"
    >
       ðŸŽ‰ All Tasks Completed! 
                            
    </h2>
    <ul
      className="c3"
    />
  </div>
</div>
`;
